R/reportPrices.R

```diff
@@ -1015,7 +1032,9 @@ reportPrices <- function(gdx, output=NULL, regionSubsetList=NULL,
   map <- data.frame(region=getRegions(out),world="GLO",stringsAsFactors=FALSE)
   tmp_GLO <- new.magpie("GLO",getYears(out),getNames(out),fill=0)
 
-  for (i2e in names(int2ext)){
+  int2ext <- int2ext[intersect(names(int2ext), getNames(out))] # select only data that exists
```

Comment:
not familiar with this. would be great if @pweigmann could check this

Reply:
looks good!